STATUS_GO_VERSION

```diff
@@ -1 +1 @@
-0.21.0g-beta.2
+0.23.0-beta.0
```

Comment:
can we use a `beta.1` instead? it has a few more fixes...

Reply:
sure